packages/app-bridge-theme/src/types/Guideline.ts

```diff
@@ -11,41 +11,48 @@ export enum LinkSettingsIconPosition {
     Left = 'LEFT',
 }
 
-export interface GuidelineCoverPage {
+interface CoverPageBase {
     id(): number;
     title(language?: string): string;
     isPublished(): boolean;
     isHiddenInNavigation(): boolean;
     url(language?: string): string;
     type: 'cover-page';
 }
+export interface CoverPageNavigationItem extends CoverPageBase {}
+export interface CoverPage extends CoverPageBase {}
 
-export interface GuidelineDocumentGroup {
+interface DocumentGroupBase {
     id(): number;
     title(language?: string): string;
-    children(): (GuidelineDocument | GuidelineDocumentLibrary | GuidelineDocumentLink)[];
+    children(): (DocumentNavigationItem | DocumentLibraryNavigationItem | DocumentLinkNavigationItem)[];
     type: 'document-group';
 }
+export interface DocumentGroupNavigationItem extends DocumentGroupBase {}
 
-export interface GuidelineDocument {
+interface DocumentBase {
     id(): number;
     title(language?: string): string;
     slug(language?: string): string;
     url(language?: string): string;
     parentId(): Nullable<number>;
```

Comment:
To me, it feels that `parentId()` makes only sense in the navigation tree context. However as a single `Document` I wonder whether we need it. And if we decide to keep it. Should we rename it here to `documentGroupId()`? This would help developers to better understand what this ID really refers to. Wdyt?

Reply:
I think `parentId` is a good name. It gives consistency to the different types.
And in fact, i believe `DocumentPage[...]` should also have a `parentId`

Reply:
@anxobotana IMO in a tree, `parentId()` is a well established and good name and I would definitely keep it there. However, when we have a single item that is not in a tree, it would make sense that the developer that uses this interface knows which entity it refers to. I would not rename the parentId() in the NavigationItem interfaces, however I believe that it would be better to have a clear name for the normal interfaces.

Reply:
and for the `DocumentPage[...]` (no navigation items) wouldn't it make sense also having a `documentId`?